src/js/gi/config.jsx

```diff
@@ -0,0 +1,21 @@
+import environment from '../common/helpers/environment';
```

Comment:
This file doesn't need the `jsx` suffix, since it doesn't use any React, so can it just be `config.js`.